test/WebJobs.Script.Tests/TestScripts/WindowsBatch/WebHookTrigger/run.bat

```diff
@@ -1,3 +1,3 @@
 ﻿echo OFF
-SET /p req=
+SET /p req=<%req%
```

Comment:
I think it takes some crazy for loops, as in [here](http://stackoverflow.com/questions/134001/how-can-i-load-the-contents-of-a-text-file-into-a-batch-file-variable).

But since it's not a new issue, let's handle separately as you suggest. A completely different approach would be to already populate an env variable with the content of the file before calling the batch file. But this only make sense for text files. so there is some trickiness.